packages/gatsby-theme-docs/src/layouts/internals/layout-main.js

```diff
@@ -23,8 +23,6 @@ const Container = styled.main`
     `}
 `;
 
-const LayoutMain = props => (
-  <Container role="main" aria-label="Page content" {...props} />
```

Comment:
`<main>` elements don't need an explicit ARIA "main" role, the element is implying the role already.

Reply:
Yes, it was mostly for backwards compatibility.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main#Browser_compatibility

Reply:
If it's just for "classic" IE I would say we can leave it out.

Reply:
Sure, you can remove the `const LayoutMain = props => <Container {...props} />;` and rename the `Container` to `LayoutMain`.